app/controllers/project_user_roles_controller.rb

```diff
@@ -8,9 +8,20 @@ def destroy
     redirect_to project_path(project)
   end
 
+  def create
+    project = find_project
+    project_user_role = project.project_user_roles.build
+    project_user_role.update(project_user_role_params)
```

Comment:
Maybe `project.project_user_roles.create(project_user_role_params)`
or
`ProjectUserRole.create(project: project, **project_user_role_params)`  (or merge project to params)